assessment.md

```diff
@@ -82,39 +82,75 @@ TODO: add description for each assessment level for this skill.
+  * :blush: = Often uses the issue tracker, but doesn't communicate enough with the team, sometimes causing confusion among the group.
+  * :laughing: = Always uses the GitHub issue tracker, and always appropriately, informing the team about their new posts to the issue tracker.
 
 * Assigning team members to issues in the GitHub issue tracker
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student clearly does not know how to assign members to issues.
+  * :worried: = Student knows how to add members to issues, but never does this appropriately.
+  * :neutral_face: = Student communicates with members sometimes, but needs to check with the team more often when assigning members to issues.
+  * :blush: = Student knows how and when to add members to issues, but sometimes does not communicate with the team on what they are doing.
+  * :laughing: = Communication and knowledge is superior.  Student's teammates are always aware and have agreed upon which issues they are going to be assigned to.
 
 * Creating and assigning labels to issues in the GitHub issue tracker
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: =
+  * :worried: =
+  * :neutral_face: =
+  * :blush: =
+  * :laughing: =
 
 * Creating and using a repository branch
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never uses repository branches.
+  * :worried: = Uses repository branches when they are not appropriate.
+  * :neutral_face: = Occasionally uses branches appropriately, but needs to use them more frequently.
+  * :blush: = Demonstrates knowledge relating to the use of branches, but isn't perfect with their use.
+  * :laughing: = Always uses branches when appropriate, and uses them frequently.
 
 * Creating and using a repository fork
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never uses repository forks.
+  * :worried: = Sometimes uses forks, but not when appropriate.
+  * :neutral_face: = Student uses forks when necessary.
+  * :blush: = The student uses forks often, but the student uses them when they are not always necessary.
+  * :laughing: = Student demonstrates superior ability to use forks and always knows when they are necessary to use.
 
 * Merging a branch or a fork to another branch or fork
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Not merging the branch or fork to another branch or fork
+  * :worried: = Merging the branch or fork when there are merge conflicts still present in the original
+  * :neutral_face: = Merging the branch or fork with no merge conflicts, but without adding reviewers to merge
+  * :blush: = Merging the branch or fork with no merge conflicts and with reviewers, but with no comments about what it is  
+  * :laughing: = Merging the branch or fork with no merge conflicts, with reviewers, and with comments about what it is
```

Comment:
I think you should replace `comments` with `description` since comments usually refer to comments made in each file containing code. Descriptions are more general and they refer to the PR description as well as other discussions that could take place. Finally, it's also worth mentioning merging branches where the continuous integration build might be failing and how that should not be ok.